logstash-core/spec/logstash/instrument/null_metric_spec.rb

```diff
@@ -8,7 +8,53 @@
     expect(described_class).to implement_interface_of(LogStash::Instrument::NamespacedMetric)
   end
 
+  let(:key) { "galaxy" }
+
+  describe "#increment" do
```

Comment:
same with these specs, most of the specs for the nullmetric are also valid for the Namespaced metric (the `.to_not raise_error`, namespacing resulting in an instance of self.class, etc)